Tests/UnitTests/Animation/Bell/Constants/BellConstantsTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BellConstantsTests.swift
+//  SparkCommon
+//
+//  Created by robin.lemaire on 25/11/2024.
+//  Copyright © 2024 Adevinta. All rights reserved.
+//
+
+import XCTest
+@testable import SparkCommon
+
+class BellConstantsTests: XCTestCase {
+
+    func testRotationInDegrees() {
+        XCTAssertEqual(
+            BellConstants.rotationInDegress,
+            10,
+            "La valeur de rotation en degrés devrait être 10"
```

Comment:
Seems weird looking at something in french when the whole project is in english

Reply:
HAHA, ADAAAAAAAAAA generated the file, it's not my fault (Ok bad excuse) ! Fixed !